examples/redshift-data-warehouse-example/README.md

```diff
@@ -0,0 +1,92 @@
+# Redashift Data Warehouse example 
+
+In this example, we build a Data Warehouse on top of a Data Lake to support large interactive SQL queries with high performance and low latency. This `README` is a step-by-step deployment guide. You can read more details about this example solution in the [documentation](https://awslabs.github.io/data-solutions-framework-on-aws/docs/examples/redshift-data-warehouse)
+
+## Pre-requisite
+
+1. Docker
+2. [Install the AWS CDK CLI](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+3. [Bootstrap the AWS account](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html)
+
+```bash
+cdk bootstrap \
+--profile <YOUR_AWS_PROFILE> \
+--cloudformation-execution-policies “POLICY_ARN” \
+aws://<ACCOUNT_ID>/<REGION>
+```
+
+## Getting started
+
+1. Download the Redshift Data Warehouse example from the Data Solutions Framework on AWS Github and unzip it:
+
+```bash
+curl -LJO https://github.com/awslabs/data-solutions-framework-on-aws/releases/latest/download/redshift-data-warehouse-example.zip
+unzip redshift-data-warehouse-example.zip -d redshift-data-warehouse-example
+cd redshift-data-warehouse-example
+```
+
+2. Modify the `./requirements.txt` to add the `cdklabs.aws_data_solutions_framework` library as a dependency:
+
+```
+aws-cdk-lib==2.114.1
```

Comment:
to avoid version mismatch, lets keep this without version and always pick the latest. We will bump in the future the CDK version and will need to come back here and bump it again, which is not desirable from a maintenance perspective. Currently there is already a mismatch between this example and the what DSF is using (2.145.0).